corehq/apps/app_manager/models.py

```diff
@@ -792,6 +792,43 @@ def short_comment(self):
         return self.comment if len(self.comment) <= 500 else self.comment[:497] + '...'
 
 
+class CaseLoadReference(DocumentSchema):
+    _allow_dynamic_properties = False
+    path = StringProperty()
+    properties = ListProperty(unicode)
```

Comment:
Just a note for a future world where all case management happens in the form builder this should also include `case_type`